From c5f498e5f07ae5f8faf3006e96e4f6e8109af4c0 Mon Sep 17 00:00:00 2001
From: Michal Oleszczyk <m.oleszczyk@grinn-global.com>
Date: Tue, 17 Oct 2017 09:22:15 +0200
Subject: [PATCH] chiliboard config: use CONFIG_DEFAULT_FDT_FILE as env variable

Remove hardcoded ftd file name from environment variables.
Use CONFIG_DEFAULT_FDT_FILE macro instead.

---
 include/configs/chiliboard.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/chiliboard.h b/include/configs/chiliboard.h
index b7887d90d4..153789e6be 100644
--- a/include/configs/chiliboard.h
+++ b/include/configs/chiliboard.h
@@ -45,7 +45,7 @@
 	"boot_fdt=try\0" \
 	"console=ttyO0,115200n8\0" \
 	"image=zImage\0" \
-	"fdt_file=am335x-chiliboard.dtb\0" \
+	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"ip_dyn=yes\0" \
 	"optargs=\0" \
 	"loadbootscript=" \
-- 
2.11.0

